src/Menu.js

```diff
@@ -17,7 +17,7 @@ module.exports = class extends React.Component {
     ambManager: PropTypes.object.isRequired
   };
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
Should we use the `constructor`, instead?

Reply:
Up to you. I'm not very familiar with this codebase and I decided to update this based on React's recommendation. ATM this will only be set on the client since this lifecycle method doesn't run on the server.

Reply:
I guess we can move this to `constructor`. @ivoreis any specific reason for it to go inside `componentDidMount` ?

Reply:
This PR goal is to update the code to be compatible with later react version. Can @davidtheclark provide more context why this wasn’t set in the constructor but in componentWillMount instead in the first place?

Reply:
@davidtheclark Any update on this ?

Reply:
I can't think of any reason.

Reply:
@ivoreis Let's movie this inside constructor ?